scripts/dev_cli.sh

```diff
@@ -155,6 +174,7 @@ cmd_help() {
     echo "        --debug               Build the debug binaries. This is the default."
     echo "        --release             Build the release binaries."
     echo "        --libc                Select the C library Cloud Hypervisor will be built against. Default is gnu"
+    echo "        --volumes             Comma separated volumes to be exported. Example --volumes /mnt:/mnt,/myvol:/myvol"
```

Comment:
I think the syntax needs to be reworked a bit, because Docker's syntax for volume is `--volume A:B,C`, where `C` is optional and a comma separated list.

You can choose a different separator here.

Reply:
Another way of doing it is to allow the script to take multiple `--volume` arguments and pass those directly to Docker.